tests/core-addon/unit/Core.test.js

```diff
@@ -30,10 +30,10 @@ describe('Core', function () {
     // Note that this will fake our data to make FAKE_STUDY_ID look
     // installed.
     chrome.management.getAll
-      .callsArgWith(0, [{type: "extension", id: FAKE_STUDY_ID}])
+      .callsArgWith(0, [{ type: "extension", id: FAKE_STUDY_ID }])
       .resolves();
     chrome.management.getAll.yields(
-      [{type: "extension", id: FAKE_STUDY_ID}]);
+      [{ type: "extension", id: FAKE_STUDY_ID }]);
```

Comment:
Most of the changes to this file are unrelated. Even if our linter is wrong, would you mind reverting it and just keep the relevant changes here?